b2_storage/backblaze_b2.py

```diff
@@ -55,11 +57,14 @@ def upload_file(self, name, content):
         }
 
         download_response = requests.post(url, headers=headers, data=content.read())
+        # Status is 503: Service unavailable. Try again
```

Comment:
It looks like sometimes servers go down (maybe related to DNS issues lately?) but the next request seems to route to a new server